source/jormungandr/jormungandr/modules/v1_routing/v1_routing.py

```diff
@@ -192,6 +193,11 @@ def setup(self):
                             region + 'isochrones',
                             endpoint='isochrones')
 
+        if app.config['HEAT_MAP']:
```

Comment:
minor: use app.config.get(), if `HEAT_MAP` is not set in the config, we don't need to chage the config